### PR TITLE
[Accessibility] Merge all accessibility changes to master.

### PR DESCRIPTION
**Pull request by veler**
Merge all accessibility changes to master.

Also in this PR : 
- Moved the "beta" logo so it's in a better tabbing order.
- Made the Facebook, Twitter and couple of other new buttons accessible.

**I tested this PR with Adafruit :**
Everything looks good except the tutorial that is fixed in one of the PR below.

**I tested this PR with Chibitronics :**
Everything looks good except the tutorial and the hidden menu that are fixed in PRs below.

## VERY IMPORTANT

**Before merging the current PR, merge the following one as it is based on the current branch :  [https://github.com/Microsoft/pxt/pull/2911](https://github.com/Microsoft/pxt/pull/2911)**

## Important

Since we changed the way the markdown is parsed, the tutorial was not starting anymore because it was not parsing the documentation correctly. We must be sure to merge the following PR from Adafruit, Chibitronics to Master at the same time :

| Target  | PR to approve and merge with the current one |
| ------------- | ------------- |
| Adafruit |  [https://github.com/Microsoft/pxt-adafruit/pull/318](https://github.com/Microsoft/pxt-adafruit/pull/318), [https://github.com/Microsoft/pxt-adafruit/pull/287](https://github.com/Microsoft/pxt-adafruit/pull/287) |
| Chibitronics  | [https://github.com/Microsoft/pxt-chibitronics/pull/66](https://github.com/Microsoft/pxt-chibitronics/pull/66), [https://github.com/Microsoft/pxt/pull/2911](https://github.com/Microsoft/pxt/pull/2911) |
| SparkFun | [https://github.com/Microsoft/pxt-sparkfun/pull/48](https://github.com/Microsoft/pxt-sparkfun/pull/48) |

**DO NOT** merge this without the PRs above.

### CODE REVIEW

**Comment by microsoftsam** [comment]
Build's not passing..

**Comment by microsoftsam** [comment]
Pull latest master.

**Comment by veler** [comment]
Build looks working now.

**Comment by microsoftsam** [comment]
Add tutorial fixes to this PR: 
https://github.com/Microsoft/pxt/commit/56b821f7e76ecf6b8e73c334f4faaacb960c1e2b
and 
https://github.com/Microsoft/pxt/commit/260057c59be34eaf6bc4f46e0ae3d20d8484a061

**Comment by veler** [comment]
As the way we read the markdown for the tutorial has changed, now the tutorial doesn't start on Adafruit. We should make the adafruit's tutorial MAS compliant too right?

**Comment by microsoftsam** [comment]
Yes. And test both changes together..

**Comment by pelikhan** [comment]
When we update Adafruit, yes -- we need to fix any tutorials

**Comment by microsoftsam** [comment]
Pull latest master. You shouldn't have changes to vs/editor files

File: theme/pxt.less [hunk 1]
```diff
@@ -7,7 +7,10 @@
 @import 'tutorial';
 @import 'sidedoc';
 
+@import 'fieldeditors';
+
```
Comment (microsoftsam) [inline theme/pxt.less]: I think this is meant to be in blockly-core.less so no need for this change.

**Comment by microsoftsam** [comment]
Even after a pull from master, I still see vs/editor files. You should jake clean, and do an npm install..

**Comment by veler** [comment]
It must be fixed now

File: theme/pxt.less (hunk 1 above)
Comment (veler) [reply theme/pxt.less]: Good catch

File: theme/theme.config [hunk 2]
```diff
@@ -43,7 +43,7 @@
 @form       : 'default';
 @grid       : 'pxt';
 @menu       : 'pxt';
-@message    : 'default';
+@message    : 'pxt';
```
Comment (microsoftsam) [inline theme/theme.config]: Pull latest of master. Isn't this already in?

Comment (veler) [reply theme/theme.config]: It's because I did a cherry pick of the branch you merged that was fixing the color contrast on "V0" ;-)

**Comment by microsoftsam** [comment]
Good work!

**Pull request merged**